tests/unit/customizations/codedeploy/test_push.py

```diff
@@ -8,19 +8,18 @@
 #
 # or in the "license" file accompanying this file. This file is
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
-# ANY KIND, either express or implied. See the License for the specific
+# mock.ANY KIND, either express or implied. See the License for the specific
```

Comment:
Find and replace got a little too aggressive :D

```suggestion
# ANY KIND, either express or implied. See the License for the specific
```

Reply:
Lol nice catch